asciimatics/widgets/frame.py

```diff
@@ -498,26 +527,26 @@ def move_to(self, x, y, h):
         :param y: The y location to make visible.
         :param h: The height of the location to make visible.
         """
-        if self._has_border:
+        if self._border_mgr.has_border:
```

Comment:
Can we use the `get_rectangle` code here too?

Reply:
Maybe not. I did this and it broke a unit test. The difference between the code here and get_rectangle is the code here uses "start_y = self._canvas.start_line + 1", where "get_rectangle()" just uses "start_y = 1".

Should get_rectangle() be using the former? What's ".start_line" for?

Reply:
`start_line` is tge first visible line in the Canvas based on how you've scrolled it thus far.  At start of day it is always zero.  As you scroll, the `start_line` changes and widgets move.  However the border is also drawn in the same Canvas and so it needs to move so that it appears unmoved to the user.  In an ideal world, I'd be using a separate Canvas/Screen to draw the border so that this translation isn't required, but somethimes life is too short! :-)

Net is that there is a translation required for the border coordinates.  You could do that here by simply adding `start_line` to the returned value.  Or possibly put that correction in  your new class?  The other case should always be used on a new Canvas and so still return what you had before...